src/main/webapp/mentors.js

```diff
@@ -83,9 +79,19 @@ export function createMentorElement(mentor) {
   if (mentor.image != null) {
     mentorCardBody.appendChild(createImg(mentor.image));
   }
-  if (mentor.name != null) {
-    mentorCardBody.appendChild(createNameHeader(mentor.name));
-  }
+
+  const mentorNameContainer = document.createElement('a');
+  mentorNameContainer.className = 'stretched-link';
+  const breakoutUrl = new URL('/mentor_breakout.html', window.location.origin);
+  console.log(mentor.userID);
+  breakoutUrl.searchParams.append('mentorID', mentor.userID);
+  mentorNameContainer.href = breakoutUrl;
+  const mentorName = document.createElement('h5');
+  mentorName.className = 'card-title text-primary';
+  mentorName.innerHTML = mentor.name;
+  mentorNameContainer.appendChild(mentorName);
+  mentorCardBody.appendChild(mentorNameContainer);
```

Comment:
Q: Is there a reason this was moved into the main function?

Reply:
Yes! Because Obi changed the backend so that all users now have a name, I figured it was best dealt with in the 'main' function.

However, if you believe this makes the code less readable, I could very easily create the mentorNameContainer in this function, and keep the creation of the mentorName element seperated into a different function.

Reply:
Okay! Thanks for the answer. I don't have any real strong opinion on this so feel free to keep it where you think it's most logical.

Reply:
I think that this new logic should be broken out into a separate function because:
- It doesn't depend much on the other logic in createMentorElement
- createMentorElement is already a large function and we want to avoid making it larger if possible.